### ISSUE TITLE
is XEP-0313 implemented completely?

### ISSUE THREAD

**Issue by jazzl0ver** [igniterealtime/openfire-monitoring-plugin#105]
Hi, 

According to the standard (paragraph 3.5):
```When a message is archived, the server MUST add an <stanza-id/> element as defined in Unique and Stable Stanza IDs (XEP-0359) [2] to the message, which informs the recipient of where and under what ID the message is stored.```

When I check logs of my jabber client, I don't see <stanza-id/> element added.

Do you have plans to implement this?

**Comment by guusdk** [comment]
Unique and Stable Stanza IDs (XEP-0359) are added (by Openfire, not the monitoring plugin) for messages that are stored in the MUC MAM archives. Sadly, that does not occur for messages that go in private/personal MAM archives.

**Comment by jazzl0ver** [comment]
> Do you have plans to implement this?

**Comment by guusdk** [comment]
I agree that it should be put in. It's a matter of finding time/resources to do it.

**Comment by jazzl0ver** [comment]
Hope, you'll find it soon! Thank you!
